tests/acceptance/features/bootstrap/TrashbinContext.php

```diff
@@ -231,21 +231,39 @@ public function listTrashbinFolderCollection(?string $user, ?string $collectionP
 			'trash-bin',
 			$davPathVersion
 		);
+		$response->getBody()->rewind();
+		$statusCode = $response->getStatusCode();
+		$respBody = $response->getBody()->getContents();
+		Assert::assertEquals("207", $statusCode, "Expected status code to be '207' but got $statusCode \nResponse\n$respBody");
+
 		$responseXml = HttpRequestHelper::getResponseXml(
 			$response,
 			__METHOD__ . " $collectionPath"
 		);
 
 		$files = $this->getTrashbinContentFromResponseXml($responseXml);
+
+		//set endpoint according to webdav request (1 = old, 2 = new, 3 = spaces)
+		$endpoint = "/remote.php/dav/trash-bin/$user";
```

Comment:
this is for new dav path. Please, add old dav path as well like with spaces

Reply:
there is no end point for listing trashbin in old dav version @saw-jan dai